libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj

```diff
@@ -29,18 +29,21 @@
     
 	<ItemGroup>
 		<PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="all" />
+		<PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.1.1" />
 		<PackageReference Include="Microsoft.Bot.Connector" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
 		<PackageReference Include="Microsoft.Bot.Connector" Condition=" '$(PackageVersion)' != '' " Version="$(PackageVersion)" />
+    <PackageReference Include="Microsoft.Bot.StreamingExtensions" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
+    <PackageReference Include="Microsoft.Bot.StreamingExtensions" Condition=" '$(PackageVersion)' != '' " Version="$(PackageVersion)" />
```

Comment:
I don't think we want this in here either.

Reply:
Let's chat about this. We're adding a dependency on Bot.StreamingExtensions to Bot.Builder, if there's a cleaner way of doing so that doesn't interfere with the various build configurations I'm happy to adopt it.